### DISCUSSION TITLE
SQL request + borrow a document.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#295] [Q&A] [answered]
Situation : I have created my personal query, it displays the list of books with the columns in the right order.
I can edit the documents (SQL query with myoid and type).

`SELECT accession_number AS "Référence", title AS "Titre", author AS "Auteur", isbn13 AS "ISBN", location AS "Localisation", deweynumber AS "Dewey", target_audience AS "Public", originality AS "Origine", myoid, type FROM book ORDER BY CAST(accession_number AS REAL) ASC`

But I can't borrow it because on the copy choice panel, no copy appears.
What more do I have to do to see the copies appear?

**Comment by textbrowser** [comment]
It doesn't fulfill the requirements.

**Comment by textbrowser** [comment, marked as answer]
SELECT accession_number AS "Référence", title AS "Titre", author AS "Auteur", isbn13 AS "ISBN", location AS "Localisation", deweynumber AS "Dewey", target_audience AS "Public", originality AS "Origine", quantity AS 'Quantity', myoid, type FROM book ORDER BY CAST(accession_number AS REAL) ASC

**Comment by textbrowser** [comment]
BQ will adapt to some incomplete queries.

**Reply by meteos77** [reply]
For my request, it works well, thank you for the evolution

**Comment by textbrowser** [comment]
BQ is aware of the shortfalls of humanity.

https://github.com/textbrowser/biblioteq/commit/bcd1205e0ffb30de1b05eec36c418e801476c2bd

**Reply by meteos77** [reply]
The release note starts to make a lot of new features for the future version.

This will probably be our version for training and the beginning of use in our library.
It's great, it has all the main features to make the transition as smooth as possible.

Thank you again for the efforts you have made over the last 18 months to make BiblioteQ a pleasure to use.

**Reply by textbrowser** [reply]
:)
